src/Components/CreateImageWizard/formComponents/Registration.js

```diff
@@ -140,13 +140,20 @@ const Registration = ({ label, ...props }) => {
     registerSystem === 'register-now-insights' ||
       registerSystem === 'register-now'
   );
+  const radioRef = useRef(null);
+
+  useEffect(() => {
+    // Set focus when the component mounts
+    if (radioRef.current) {
+      radioRef.current.focus();
+    }
+  }, []);
```

Comment:
Why adding a `useEffect` change the focus here ? At first glance it looks unrelated to linting warning fixes. Can you develop why this is needed please?

Reply:
becasue I remove the autofocus that bring this warning:
  150:13  warning  The autoFocus prop should not be used, as it can reduce usability and accessibility for users  jsx-a11y/no-autofocus

  const radioRef = useRef(null);

  useEffect(() => {
    // Set focus when the component mounts
    if (radioRef.current) {
      radioRef.current.focus();
    }
  }, []);
  and one test was fail because the we still need that focus

Reply:
Quick note on the auto focus - I had a short discussion about it with a few people some months ago and the outcome was that a11y's "auto focus bad" approach is not really always true as it's quite heavily contextual. 

Accessibility is one of the areas we should and want to look into more as it was not a focus until now. But generally I'd say let's keep the warning for now as a reminder and deal with it when we get to the topic. Adding a11y rules was just a beginning and the rule was switched from error to warning for this purpose. :)

Reply:
remove the usefref from that pr for now